vowpalwabbit/gdb_pretty_printers.py

```diff
@@ -1,5 +1,7 @@
-import gdb
+"""GDB Pretty Printing Module."""
```

Comment:
are the comments in this file necessary?

Reply:
Yes. These are docstrings and as per good linting standards, each module, method and function should have a docstring.

Reply:
This file isn't part of the vowpalwabbit package, and the comments seem quite redundant.

Reply:
Okay. I believe I should remove the docstrings then? Shall I?

Reply:
Yes

Reply:
@lalo @jackgerrits Thanks for the review. The redundant docstrings have been removed.